src/LogHandler.ts

```diff
@@ -0,0 +1,72 @@
+interface LogFunction {
+	(message: string): void;
+}
+
+export interface Logger {
```

Comment:
I had thought that this complied with the npm log levels (https://github.com/npm/npmlog) and I knew that winston (https://www.npmjs.com/package/winston) was a popular library that offered npm log level support.  However, upon closer inspection, there is an `http` log level which isn't represented here.  That level feels a bit weird, but I suppose it could be thrown in and used to log request information to MVIS admin and MVIS.

Reply:
I guess technically the npm log levels _do_ conform to this interface.  It just happens to have an extra log level of `http` which isn't used.  So you could provide something with the npm log levels to this and have it work fine.